lib/schema-evolution-manager/apply_util.rb

```diff
@@ -35,6 +36,24 @@ def apply!(dir)
       count
     end
 
+    def persist_pgpass(password)
+      Preconditions.check_not_blank(password, "password cannot be blank")
+      puts "Creating temp pgpass at #{@pgpass_file.path}"
+      FileUtils.chmod(0600, @pgpass_file.path)
+
+      FileUtils.rm_rf("#{ENV['HOME']}/.pgpass")
```

Comment:
I agree it is a valid concern. 

I added a warning to the README, maybe I should also add a warning step before entering the password. I avoided going down the path of PGPASSWORD because of this comment:

http://www.postgresql.org/docs/9.1/static/libpq-envars.html

`PGPASSWORD behaves the same as the password connection parameter. Use of this environment variable is not recommended for security reasons, as some operating systems allow non-root users to see process environment variables via ps; instead consider using the ~/.pgpass file (see Section 31.14).`